apps/meteor/server/models/raw/Permissions.ts

```diff
@@ -61,7 +61,15 @@ export class PermissionsRaw extends BaseRaw<IPermission> implements IPermissions
 		await this.updateOne({ _id: permission, roles: role }, { $pull: { roles: role } });
 	}
 
+	findById(_id: IPermission['_id'], options?: FindOptions<IPermission>): Promise<IPermission | null> {
```

Comment:
not sure if I'm missing something, but why would you use this one instead of `findOneById` ?